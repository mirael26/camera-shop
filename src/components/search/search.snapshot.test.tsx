import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import Search from './search';

jest.mock('./search-list/search-list', () => 'SearchList');

test('Search component renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<Search/>);
  expect(asFragment()).toMatchSnapshot();
});
