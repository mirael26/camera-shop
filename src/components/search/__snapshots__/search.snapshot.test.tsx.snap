// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component renders correctly 1`] = `
<DocumentFragment>
  <div
    class="form-search"
    data-testid="form-search"
  >
    <form>
      <label>
        <svg
          aria-hidden="true"
          class="form-search__icon"
          height="16"
          width="16"
        >
          <use
            xlink:href="#icon-lens"
          />
        </svg>
        <input
          autocomplete="off"
          class="form-search__input"
          placeholder="Поиск по сайту"
          type="text"
          value=""
        />
      </label>
    </form>
    <button
      class="form-search__reset"
      type="reset"
    >
      <svg
        aria-hidden="true"
        height="10"
        width="10"
      >
        <use
          xlink:href="#icon-close"
        />
      </svg>
      <span
        class="visually-hidden"
      >
        Сбросить поиск
      </span>
    </button>
  </div>
</DocumentFragment>
`;
